src/HomeKit/HMEnums.cs

```diff
@@ -411,6 +411,14 @@ public enum HMCharacteristicType {
 		[Field ("HMCharacteristicTypeImageMirroring")]
 		ImageMirroring,
 
+		[iOS (10,3), Watch (3,2), TV (10,2)]
+		[Field ("HMCharacteristicTypeLabelNamespace")]
+		LabelNamespace,
+
+		[iOS (10,3), Watch (3,2), TV (10,2)]
+		[Field ("HMCharacteristicTypeLabelIndex")]
+		LabelIndex,
+
```

Comment:
we missed it but that's incorrect - you cannot add enum values (without numbering them) in the middle of others without breaking all the one that comes after, see API diff results